integration-tests/integration_tests/test_scenarios/int_async_express_tests.py

```diff
@@ -82,13 +81,5 @@ def test_async_express_sync_async_wrap(self):
                         "Async Express outbound test failed")
 
         root = ET.ElementTree(ET.fromstring(outbound_response.text)).getroot()
-
-        self.assertTrue(self.does_contain_success_code(root))
-
-    def does_contain_success_code(self, root):
-        for child in root.iter():
-            if child.tag == '{urn:hl7-org:v3}queryResponseCode':
-                self.assertEqual(child.attrib['code'], 'OK')
-                return True
-
-        return False
+        element = list(root.iter('{urn:hl7-org:v3}queryResponseCode'))[0]
```

Comment:
I tried fiddling with this, and eventually figured it out (with some [Googling](https://stackoverflow.com/a/55049369)):
```suggestion
        element = root.find('.//hl7:queryResponseCode', namespaces={'hl7': 'urn:hl7-org:v3'})
```
Have to use XPath as without it, it's a non-recursive find.